Session.js

```diff
@@ -0,0 +1,54 @@
+// To be utilized when unwrapping JSON from the future servlets. Will be on the 
+// client side and be used as a buffer for client to server information needed  
+// for other components of Virtual Movie Night.
+class Session { 
```

Comment:
also check spacing, i believe JS is only a two vertical space required for indentation in these methods/constructors